src/Controllers/SchemaController.js

```diff
@@ -151,10 +151,16 @@ const defaultColumns: { [string]: SchemaFields } = Object.freeze({
 });
 
 const requiredColumns = Object.freeze({
-  _Product: ['productIdentifier', 'icon', 'order', 'title', 'subtitle'],
-  _Role: ['name', 'ACL'],
+  read: {
+    _User: ['username'],
+  },
+  write: {
+    _Product: ['productIdentifier', 'icon', 'order', 'title', 'subtitle'],
+    _Role: ['name', 'ACL'],
+  }
 });
 
+
```

Comment:
could you remove that 2nd empty line? lint should probably complain about multiple empty lines